app/views/install/compose.phtml

```diff
@@ -397,7 +395,6 @@ services:
     depends_on:
       - redis
       - mariadb
-      - appwrite-executor
```

Comment:
shouldn't it now depend on openrutntimes-executor?

Reply:
Re-added.